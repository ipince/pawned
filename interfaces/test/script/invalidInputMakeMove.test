# Invalid inputs in MakeMove command
StartNewGame human 0 human 0
IsLegalMove a2-a3
MakeMove
MakeMove a2-a3
MakeMove 0 a2-a3
MakeMove a2-a3 a
MakeMove a a2-a3
MakeMove 0
MakeMove a2-a3 0 0
MakeMove a2-a3 0 b2-b3
MakeMove a2-a3 -10
StartNewGame human 1000 human 1000
IsLegalMove a2-a3
MakeMove
MakeMove a2-a3
MakeMove 10 a2-a3
MakeMove a2-a3 a
MakeMove a a2-a3
MakeMove 10
MakeMove a2-a3 10 10
MakeMove a2-a3 10 b2-b3
MakeMove a2-a3 -100