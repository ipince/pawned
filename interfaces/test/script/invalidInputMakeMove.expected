# Invalid inputs in MakeMove command
New game started
legal
Input error
Input error
Input error
Input error
Input error
Input error
Input error
Input error
Input error
New game started
legal
Input error
Input error
Input error
Input error
Input error
Input error
Input error
Input error
Input error
